deploy.config.js

```diff
@@ -29,10 +29,19 @@ exports.providers = {
             incentiveToken: "0x0000000000000000000000000000000000000000",
         },
         alfajores: {
-            lendingPoolAddressProvider: "0x6EAE47ccEFF3c3Ac94971704ccd25C7820121483",
-            lendingPool: "0x0886f74eEEc443fBb6907fB5528B57C28E813129",
+            lendingPoolAddressProvider: "0xb3072f5F0d5e8B9036aEC29F37baB70E86EA0018",
+            dataProvider: "0x31ccB9dC068058672D96E92BAf96B1607855822E",
```

Comment:
@viraj124 
wondering if it wouldn't be better to move these configs under a `moola` object, instead of `aave`. Maybe we can leave it for now, but should do that soon.

Reply:
Yeah merging for now